### PR TITLE
added channel updating operation

### PR DESCRIPTION
**Pull request by zhuyuanmao**
add an update method to channel class to achieve updating operation such as add org to the channel.

Signed-off-by: Yuanmao Zhu <zhu.yuanmao18@gmail.com>

### CODE REVIEW

File: src/api-engine/api/lib/peer/channel.py [hunk 1]
```diff
@@ -36,3 +36,13 @@ def list(self):
             err_msg = "get channel list failed for {}!".format(e)
             raise Exception(err_msg)
         return res
+
+    def update(self, channel, channel_tx, orderer_url, time_out="90s"):
+        try:
+            res = os.system("{} channel create -c {}  -f {} -o {} --timeout {}"
```
Comment (yeasy) [inline src/api-engine/api/lib/peer/channel.py:42]: Why it's "create"?

**Review by yeasy — CHANGES_REQUESTED** [review]

File: src/api-engine/api/lib/peer/channel.py [hunk 2]
```diff
@@ -37,12 +37,38 @@ def list(self):
...
+
+        params:
+            channel: channel id.
+            channel_tx: Configuration transaction file generated by a tool such as configtxgen for submitting to orderer
+            orderer_url: Ordering service endpoint.
+        """
         try:
-            res = os.system("{} channel create -c {}  -f {} -o {} --timeout {}"
-                            .format(self.peer, channel, channel_tx, orderer_url, time_out))
+            res = os.system("{} channel update -c {}  -f {} -o {}"
```
Comment (yeasy) [inline src/api-engine/api/lib/peer/channel.py:50]: There is no such `channel update` cmd.

Comment (zhuyuanmao) [reply src/api-engine/api/lib/peer/channel.py:50]: Sorry, I am a little bit confused about it, I implemented it by the following [instructions](https://hyperledger-fabric.readthedocs.io/en/release-2.2/commands/peerchannel.html#peer-channel-update), it is `peer channel update` cmd and `self.peer` attribute indicates the path to peer CLI.

Comment (yeasy) [reply src/api-engine/api/lib/peer/channel.py:50]: oh, my bad. 
got your code now.

**Review by yeasy — APPROVED** [review]

**Pull request merged**